### DISCUSSION TITLE
Centering Header When Wider Than Global Width

### DISCUSSION THREAD

**Discussion by jackplus-xyz** [folke/snacks.nvim#108] [Q&A] [answered]
Hi everyone,

I’m experiencing an issue related to centering the dashboard header when its width exceeds the global width setting. Here’s the situation:
	•	If the header is wider than the global width, it cannot be centered properly because the block width is smaller, causing the header to overflow to the right side.
	•	Setting the global width to be larger than the header’s width will resolve the centering issue, but since the other items (e.g., keys and icons) are distributed evenly across the wider width, making them appear too far apart from each other.

Is it possible to have separate width for the header and items block? I've tried:

1. Set a smaller width for `items` but it doesn't seem to be working. 
2. I also tried to create two panes but since they are distributed evenly and split horizontally, it doesn't work either. 

Here's the header I'm using:
```lua
  {
        preset = {
          header = [[
                                                                    
       ████ ██████           █████      ██                    
      ███████████             █████                            
      █████████ ███████████████████ ███   ███████████  
     █████████  ███    █████████████ █████ ██████████████  
    █████████ ██████████ █████████ █████ █████ ████ █████  
  ███████████ ███    ███ █████████ █████ █████ ████ █████ 
 ██████  █████████████████████ ████ █████ █████ ████ ██████
]],
  },   
},
```

Here are screenshots of the issue:

1. `width` set to 60(default):
<img width="1454" alt="Screenshot 2024-11-18 at 19 45 31" src="https://github.com/user-attachments/assets/186e5328-316b-4f6f-9836-aefbcb92e383">

The header is not centered since it's starting point can be lefter than the block.

2. `width` set to 80(larger than the header width):
<img width="1454" alt="Screenshot 2024-11-18 at 19 45 44" src="https://github.com/user-attachments/assets/d7cc0502-7bf2-4ff1-8562-22b394283bda">

The items block is too wide.

**Comment by folke** [comment, marked as answer]
Fixed now
